### ISSUE TITLE
rule declaration-colon-newline-after is not proper when colon followed by function call

### ISSUE THREAD

**Issue by dyf19118** [stylelint/stylelint#2794]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

New option needed i think.
For example, provide an except option for function call.

> Which rule, if any, is this issue related to?

declaration-colon-newline-after

> What CSS is needed to reproduce this issue?

with postcss-cssnext syntax.

```css
background-image: image-set(
	url(../img/a.jpg) 1x,
	url(../img/b.jpg) 2x,
	url(../img/c.jpg) 3x
);
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "declaration-colon-newline-after": "always-multi-line",
    "declaration-colon-space-after": "always-single-line"
  }
}
```

> Which version of stylelint are you using?

`^8.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

as a PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

After adding a new option(except option), No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
src/test.css
53:18   ‼  Expected newline after ":" with a multi-line declaration (declaration
-colon-newline-after) [stylelint]
```

**Comment by evilebottnawi** [comment]
@dyf19118 rule `declaration-colon-newline-after` works as expected, your have multi line declaration. Seems your want to use inconsistent stylistics:
1. Basic:
```css
box-shadow:
    0 0 0 1px #5b9dd9,
    0 0 2px 1px rgba(30, 140, 190, 0.8);
```
2. For `image-set` other:
```css
background-image: image-set(
	url(../img/a.jpg) 1x,
	url(../img/b.jpg) 2x,
	url(../img/c.jpg) 3x
);
```
Right?

**Comment by dyf19118** [comment]
@evilebottnawi Um. I just think function call is a special case for this rule, since function parameters are split into multiple lines for readability. And is not the same like multi-line properties. Anyway, thanks for your suggestion.
